diff --git a/node_modules/@deck.gl/layers/dist/esm/solid-polygon-layer/solid-polygon-layer.js b/node_modules/@deck.gl/layers/dist/esm/solid-polygon-layer/solid-polygon-layer.js
index af90eae..fbe8f1b 100644
--- a/node_modules/@deck.gl/layers/dist/esm/solid-polygon-layer/solid-polygon-layer.js
+++ b/node_modules/@deck.gl/layers/dist/esm/solid-polygon-layer/solid-polygon-layer.js
@@ -294,6 +294,9 @@ export default class SolidPolygonLayer extends Layer {
       const {
         polygonTesselator
       } = this.state;
+
+      if (polygonTesselator.instanceCount) return
+
       const buffers = props.data.attributes || {};
       polygonTesselator.updateGeometry({
         data: props.data,
diff --git a/node_modules/@deck.gl/layers/src/solid-polygon-layer/solid-polygon-layer.ts b/node_modules/@deck.gl/layers/src/solid-polygon-layer/solid-polygon-layer.ts
index 12e08f8..2913923 100644
--- a/node_modules/@deck.gl/layers/src/solid-polygon-layer/solid-polygon-layer.ts
+++ b/node_modules/@deck.gl/layers/src/solid-polygon-layer/solid-polygon-layer.ts
@@ -371,6 +371,11 @@ export default class SolidPolygonLayer<DataT = any, ExtraPropsT = {}> extends La
     // tessellator needs to be invoked
     if (geometryConfigChanged) {
       const {polygonTesselator} = this.state;
+
+      // BILLY - don't update geometry if we already HAVE geometry
+      // (crazy slow on CPU)
+      if (polygonTesselator.instanceCount) return
+
       const buffers = (props.data as any).attributes || {};
       polygonTesselator.updateGeometry({
         data: props.data,
