diff --git a/node_modules/shp-write/src/download.js b/node_modules/shp-write/src/download.js
index d198add..af96b9c 100644
--- a/node_modules/shp-write/src/download.js
+++ b/node_modules/shp-write/src/download.js
@@ -1,6 +1,6 @@
 var zip = require('./zip');
 
-module.exports = function(gj, options) {
-    var content = zip(gj, options);
+module.exports = async function(gj, options) {
+    var content = await zip(gj, options);
     location.href = 'data:application/zip;base64,' + content;
 };
diff --git a/node_modules/shp-write/src/zip.js b/node_modules/shp-write/src/zip.js
index b8e89d8..e1cabe9 100644
--- a/node_modules/shp-write/src/zip.js
+++ b/node_modules/shp-write/src/zip.js
@@ -28,11 +28,8 @@ module.exports = function(gj, options) {
         }
     });
 
-    var generateOptions = { compression:'STORE' };
+    var generateOptions = { type: "base64"};
 
-    if (!process.browser) {
-      generateOptions.type = 'nodebuffer';
-    }
-
-    return zip.generate(generateOptions);
+    // return a PROMISE -- need to await or .then() the results
+    return zip.generateAsync(generateOptions)
 };
