diff --git a/node_modules/vuera/dist/vuera.cjs.js b/node_modules/vuera/dist/vuera.cjs.js
index afd2414..900f6ff 100644
--- a/node_modules/vuera/dist/vuera.cjs.js
+++ b/node_modules/vuera/dist/vuera.cjs.js
@@ -383,7 +383,11 @@ var ReactWrapper = {
         this.reactComponentRef.setState(_extends({}, this.$attrs));
       },
 
-      deep: true
+      // deep:true causes Deck.gl to deep-copy the entire dataset
+      // any time any attribute changes. This is BAD.
+      // Deck.gl does a good job of sensing changes.
+      // deep:false seems to work just fine. And 10000% faster on large datasets.
+      deep: false
     },
     '$props.component': {
       handler: function handler(newValue) {
@@ -408,6 +412,11 @@ var ReactWrapper = {
 };
 
 function isReactComponent(component) {
+  // BILLY: is it a promise? : then it's a vue async component
+  // This is the only way I've figured out to identify these AsyncComponents
+  // (which are definitely not React components!)
+  if (component.name === '') return false
+
   if ((typeof component === 'undefined' ? 'undefined' : _typeof(component)) === 'object' && !isReactForwardReference(component)) {
     return false;
   }
